ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/private/cucushift-installer-rehearse-gcp-ipi-private-workflow.yaml

```diff
@@ -0,0 +1,20 @@
+workflow:
+  as: cucushift-installer-rehearse-gcp-ipi-private
+  steps: 
+    pre:
+      - chain: cucushift-installer-rehearse-gcp-ipi-private-provision
+    post:
+      - chain: cucushift-installer-rehearse-gcp-ipi-private-deprovision
+    env:
```

Comment:
We are moving the env setting into the provision/deprovision  chains, that will help other team integration easier.